.sass-lint.yml

```diff
@@ -11,7 +11,7 @@ rules:
   no-url-protocols: 0
   class-name-format:
     - 1
-    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,2}$
+    - convention: ^([a-z0-9]+([A-Z][a-z0-9]+)*)(-{1}([a-z0-9]+([A-Z][a-z0-9]+)*)){0,4}$
     - convention-explanation: 'Classes should be camelCase and separated by no more than 5 hyphens'
```

Comment:
taking it to 5 reduced our error count from 109 to 106. But i agree i think we should have a larger discussion around this rule.